packages/itwinui-react/tsconfig.test.json

```diff
@@ -6,6 +6,7 @@
     "src/react-table/*",
     "src/core/Select/**/*",
     "src/core/LabeledSelect/**/*",
-    "src/core/ComboBox/**/*"
+    "src/core/ComboBox/**/*",
+    "**/*.test.?(c|m)[jt]s?(x)"
```

Comment:
Can't we simplify this entire thing as follows?

```js
"include": [
   "src/**/*.test.*",
   "src/**/*.types-test.*"
]
```

cc @rohan-kadkol

Reply:
Yes, sounds good.

The only difference is that we will now not be type testing non-test files in the components from the current `include` (E.g. type errors in `src/core/Table/actionHandlers/*.ts`, etc.). But I feel this can be handled later.

Reply:
We can add it to the PR description as a TODO item after this PR.

Reply:
Personally I feel it's unnecessary. Any non-test files should already be type checked when they are converted to `.js` during `build` step.